src/config/configStore.ts

```diff
@@ -290,35 +243,54 @@ export abstract class BaseConfigStore<
    */
   public async awaitEach(actionFn: (key: string, value: ConfigValue) => Promise<void>): Promise<void> {
     const entries = this.entries();
-    for (const entry of entries) {
-      // prevent ConfigFile collision bug
-      // eslint-disable-next-line no-await-in-loop
-      await actionFn(entry[0], entry[1]);
-    }
+    await Promise.all(entries.map((entry) => actionFn(entry[0], entry[1])));
```

Comment:
is `awaitEach` still used by anyone? I remember adding this years ago for something in toolbelt's version of SDR

Reply:
doesn't look like it, will delete

https://github.com/search?q=%28org%3Aforcedotcom+OR+org%3Asalesforcecli%29+awaitEach&type=code